tests/Main.hs

```diff
@@ -635,6 +635,7 @@ runClashTest = defaultMain $ clashTestRoot
         , outputTest "T2325f" def{hdlTargets=[VHDL]}
         , runTest "T2342A" def{hdlSim=[]}
         , runTest "T2342B" def{hdlSim=[]}
+        , runTest "T2360" def{hdlSim=[]}
```

Comment:
Shouldn't this pass `-fclash-force-undefined=0` to properly test the issue?

Reply:
And it also doesn't actually run it in any HDL tool? I think the problem only crops up on actual running?

Reply:
Shh, _you saw nothing_